Makefile

```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= quay.io/fpaoline/metallboperator:latest
+IMG ?= controller:latest
```

Comment:
this is just revert the change to hardcode the image w/o I can't test as it was always loading ur image and skip the new controller

Reply:
Let's wait for an official repo to replace this value. I asked for a new u/s image.

If you want to run custom builds to test manually you can do `IMG=whateveryouwant make deploy` and it will override the IMG variable

Reply:
yeah this is what I am doing but for that to work I have to remove the hardcode image in the manager.yaml

Reply:
revert it since PR #15 will takecare of cleaning this up